assets/tailwind.config.js

```diff
@@ -1,16 +1,22 @@
 module.exports = {
+  darkMode: false, // or 'media' or 'class'
+  important: true,
+  plugins: [],
   purge: [
     '../lib/**/*.ex',
     '../lib/**/*.leex',
     '../lib/**/*.eex',
     './js/**/*.js'
   ],
-  darkMode: false, // or 'media' or 'class'
   theme: {
-    extend: {},
+    extend: {
+      boxShadow: {
+        'inner-input': 'inset -4px -4px 10px #323637, inset 4px 4px 10px #080a0a',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/dj-rumble/dj-rumble/pullRequest?prid=7326091)
```suggestion
        "inner-input": 'inset -4px -4px 10px #323637, inset 4px 4px 10px #080a0a',
```